src/Utilities/Compiler/Analyzer.Utilities.projitems

```diff
@@ -41,6 +41,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)CodeMetrics\CodeMetricsAnalysisContext.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\..\nuget\PerformanceSensitiveAnalyzers\PerformanceSensitiveAttribute.cs" Link="PerformanceSensitiveAttribute.cs" />
```

Comment:
Hmm, can we actually dogfood the analyzer package instead of linked file like this?

Reply:
I'm not sure since it's part of a shared project. I can try...

Reply:
Updated to reference PerformanceSensitiveAnalyzers instead

Reply:
Thanks!